### ISSUE TITLE
Could DeepIV estimator generate the constant marginal effect?

### ISSUE THREAD

**Issue by yqc924** [microsoft/EconML#368]
Hi,

I am sorry for raising this question. I am just wondering, would it be possible to get the **constant** marginal effect using the DeepIV estimator? Does anyone have any idea about this? Thank you!

**Comment by kbattocchi** [comment]
No, unlike our linear estimators, DeepIV doesn't assume a constant effect, which is why that method is not present on the deep IV class.

**Comment by yqc924** [comment]
> No, unlike our linear estimators, DeepIV doesn't assume a constant effect, which is why that method is not present on the deep IV class.

Got it. Thank you very much for your response!
